outputs.tf

```diff
@@ -161,3 +161,102 @@ output "helm_releases" {
   description = "Map of attributes of the Helm release created"
   value       = helm_release.this
 }
+
+################################################################################
+# GitOps Bridge
```

Comment:
lets add some more comments on this to make it clear why this is here, what its used for, our intentions, and our gurantees

Reply:
I added comment here